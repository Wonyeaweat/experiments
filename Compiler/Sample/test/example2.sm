program example2;
var  A, B, C:integer;
begin
  A:=1; B:=0; C:=1;
  if x < 100 or x > 200 and x <> y then
     C := 1
end.