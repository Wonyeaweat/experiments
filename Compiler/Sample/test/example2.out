( 0) (program, example2, - , - )
( 1) (:=, 1 , - , A )
( 2) (:=, 0 , - , B )
( 3) (:=, 1 , - , C )
( 4) (j<, x , 100, 10)
( 5) (j , - , - , 6 )
( 6) (j>, x , 200, 8 )
( 7) (j , - , - , 11)
( 8) (j<>, x , y , 10)
( 9) (j , - , - , 11)
(10) (:=, 1 , - , C )
(11) (sys, - , - , - )
